### PR TITLE
add seealsoContent for udp

### PR DESCRIPTION
**Pull request by superyyrrzz**

### CODE REVIEW

**Comment by hellosnow** [comment]
:shipit:

File: src/Microsoft.DocAsCode.DataContracts.Common/Constants.cs [hunk 1]
```diff
@@ -48,6 +48,7 @@ public static class PropertyName
             public const string ExtensionMethods = "extensionMethods";
             public const string Overload = "overload";
             public const string Return = "return";
+            public const string SeeAlsoContent = "seealsoContent";
```
Comment (vicancy) [inline src/Microsoft.DocAsCode.DataContracts.Common/Constants.cs]: seeAlsoContent?

File: src/Microsoft.DocAsCode.DataContracts.Common/Constants.cs [hunk 2]
```diff
@@ -48,6 +48,7 @@ public static class PropertyName
...
 
             public const string SystemKeys = "_systemKeys";
```
Comment (vicancy) [inline src/Microsoft.DocAsCode.DataContracts.Common/Constants.cs]: is _systemKey used in udp?

File: src/Microsoft.DocAsCode.DataContracts.Common/Constants.cs (hunk 1 above)
Comment (superyyrrzz) [inline src/Microsoft.DocAsCode.DataContracts.Common/Constants.cs]: want to be consistent with seealso

---
In reply to: [114701331](https://github.com/dotnet/docfx/pull/1625#discussion_r114701331) [](ancestors = 114701331)

File: src/Microsoft.DocAsCode.DataContracts.Common/Constants.cs (hunk 2 above)
Comment (superyyrrzz) [inline src/Microsoft.DocAsCode.DataContracts.Common/Constants.cs]: not used now. can add it later

---
In reply to: [114701341](https://github.com/dotnet/docfx/pull/1625#discussion_r114701341) [](ancestors = 114701341)

**Pull request merged**